### ISSUE TITLE
Contact forward e-mail adress

### ISSUE THREAD

**Issue by TorstenBlochwitz** [modelica/fmi-standard.org#285]
The e-mail address projectleader@fmi-standard.org is forwarded to the project leader and his deputy. The latter points to my old e-mail address Blochwitz@itisim.com.

This needs to be changed to Torsten.Blochwitz@esi-group.com since forwarding from the old address will end soon.

**Comment by TorstenBlochwitz** [comment]
@andreas-junghanns: You might have the same proble?

**Comment by andreas-junghanns** [comment]
Yes, I have the same issue: please change to andreas.junghanns@synopsys.com.

However, I think this is the wrong repo.

Maybe https://github.com/modelica/BackOffice is better?
Or https://github.com/modelica/MA-Internal?

@t-sommer ?

**Comment by t-sommer** [comment]
Yes, please move this to https://github.com/modelica/BackOffice.

**Comment by andreas-junghanns** [comment]
> Yes, please move this to https://github.com/modelica/BackOffice.

I cannot see the Backoffice repo in the transfer list - I might not have the right.
Leo, can you do that?
